src/Merge/ExtraPackage.php

```diff
@@ -107,7 +115,12 @@ public function getRequires()
      */
     protected function readPackageJson($path)
     {
-        $file = new JsonFile($path);
+        if(substr($path, 0, 4) === 'http'){
```

Comment:
This probably should use a preg_match for `^https?://` instead of just grabbing anything starting with "http". As written an import of "http_module/composer.json" would get slurped up to be treated as a remote file.